### DISCUSSION TITLE
Initial PR not created

### DISCUSSION THREAD

**Discussion by mbmx10** [renovatebot/tutorial#22] [Q&A]
I followed the steps for a cloned github repository a while ago and everything worked perfectly.

Today I tried it again from scratch and didn't see the initial PR. Am I missing something?

**Comment by rarkins** [comment]
Do you see anything relevant on app.renovatebot.com?

**Reply by mbmx10** [reply]
Thanks for the quick response. The logs on app.renovatebot.com seem normal, and I don't see anything else which might be relevant.

The PR has now been created  - roughly 10 hours after setting up the github app.
I merged the PR and saw the updated logs. However, the dashboard has not been visible for 10 minutes now.
